scripts/clean-changelog-unreleased.mjs

```diff
@@ -2,13 +2,13 @@
 
 import fs from "node:fs";
 import { fileURLToPath } from "node:url";
-import globby from "globby";
+import { globbySync } from "globby";
 
 const changelogUnreleasedDir = fileURLToPath(
   new URL("../changelog_unreleased", import.meta.url)
 );
 
-const files = globby.sync(["blog-post-intro.md", "*/*.md"], {
+const files = globbySync(["blog-post-intro.md", "*/*.md"], {
```

Comment:
Globby accepts URL as `cwd` now.

Reply:
Actually this is the reason I propose https://github.com/sindresorhus/globby/issues/176